<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TouristBRDialog</class>
 <widget class="QDialog" name="TouristBRDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tourist_info">
       <row>
        <property name="text">
         <string>读者ID</string>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </row>
       <row>
        <property name="text">
         <string>借阅/归还时间</string>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </row>
       <row>
        <property name="text">
         <string>借阅/归还地点</string>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </row>
       <row>
        <property name="text">
         <string>借阅时长（归还时不填）</string>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </row>
       <column>
        <property name="text">
         <string>游客信息</string>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <item row="0" column="0">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="1" column="0">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="2" column="0">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="3" column="0">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="borrow_choose">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>借阅</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="return_choose">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>归还</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="OK">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icons/icons/ok.ico</normaloff>:/icons/icons/ok.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Cancel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icons/icons/cancel.ico</normaloff>:/icons/icons/cancel.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
